### ISSUE TITLE
embed-responsive does not resize youtube thumbnail for browsers that do not support video playback

### ISSUE THREAD

**Issue by bruse** [twbs/bootstrap#15981]
When trying to make an embedded youtube video responsive using the embed-responsive helper classes, the thumbnail for the video is malformed on browsers that do not support video playback in the browser (such as Opera Mini). Instead of resizing the thumbnail together with the iframe, the image gets cut off and the play-button is no longer centered (and in some cases completely hidden).

There is a minimal test case for this here: http://jsbin.com/depogumuqe/1/

Here are screenshots displaying the problem in Opera Mini 7.6.40234 on Android 4.4.4.

Using only a regular youtube embed iframe, the whole thumbnail gets displayed:
![screenshot_2015-03-03-00-40-09](https://cloud.githubusercontent.com/assets/406433/6453543/5e7101ac-c13f-11e4-9e11-febcb994c9d4.png)

Using the embed-response helper classes, the image gets cut off:
![screenshot_2015-03-03-00-42-30](https://cloud.githubusercontent.com/assets/406433/6453544/5e7574b2-c13f-11e4-90f0-f7f1d547a3fc.png)

For me, the largest issue with this is when the play button is hidden completely, removing any indication that a picture that is displayed in a webpage is in fact a clickable link to a video.

**Comment by twbs-lmvtfy** [comment]
Hi @bruse!

You appear to have posted a live example (http://jsbin.com/depogumuqe/1/edit), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 12, column 163: The ```` frameborder ```` attribute on the ```` iframe ```` element is obsolete. Use CSS instead.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Note: Opera Mini is not officially supported by Bootstrap.

**Comment by bruse** [comment]
Ah, I was not aware of this. I guess this should be closed then, if it's not some obvious easy fix that still makes it worth it.

**Comment by cvrebert** [comment]
@mdo Shall we close this then?

**Comment by mdo** [comment]
Yeah, punting as a can't/won't fix.
